artisoptions_kilonova_lte.h

```diff
@@ -146,6 +144,8 @@ constexpr float RPKT_BOUNDBOUND_THERMALISATION_PROBABILITY = -1.;
 
 constexpr bool USE_XCOM_GAMMAPHOTOION = false;
 
+constexpr auto PARTICLE_THERMALISATION_SCHEME = ThermalisationScheme::INSTANT;
```

Comment:
To preserve the exiting behaviour, the thermalisation scheme should be ThermalisationScheme::DETAILED. All other option files need to be updated to use the existing instant mode. (small detail, maybe we can switch to Title case like Instant, Detailed, BarnesGlobal, BarnesLocal).

Reply:
Yes, thanks. Will push that along with changes to your comment above.